api/provider/aws/service_create.go

```diff
@@ -12,8 +12,6 @@ func (s *ServiceProvider) Create(req models.CreateServiceRequest) (string, error
 	fqEnvironmentID := addLayer0Prefix(s.Config.Instance(), req.EnvironmentID)
 	cluster := fqEnvironmentID
 
-	desiredCount := 1
```

Comment:
is this a terraform-specific change or just a fix in general?

Reply:
fix in general